lib/src/pie_menu_controller.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:pie_menu/src/pie_menu_event.dart';
+
+/// Controller for programmatically emitting [PieMenu] events.
+class PieMenuController extends ValueNotifier<PieMenuEvent> {
+  PieMenuController() : super(PieMenuCloseEvent());
+
+  void openMenu({
+    Alignment? menuAlignment,
```

Comment:
I put these as nullable parameters because they could be available in PieTheme instead, and so are not required here. Assuming I've read your comments correctly and that's what you were after.

If you merge this branch in, feel free to change this around or delete it as you need.